README.md

```diff
@@ -91,30 +91,37 @@ Browser Mod has a number of services you can call to cause things to happen in t
 
 ## Popup card
 
-A popup card can be used to replace the more-info dialog of an entity with something of your choosing.
+A popup card can be used to replace the more-info dialog of an entity with something of your choosing. The entity can be targeted by entity, area, label or device.
 
-To use it, add a "Custom: Popup card" to a dashboard view via the GUI, pick the entity you want to override, configure the card and set up the popup like for the [`browser_mod.popup` service](documentation/services.md).
+To use it, add a "Custom: Popup card" to a dashboard view via the GUI, pick the target (entity, area, label, device) you want to override, configure the card and set up the popup like for the [`browser_mod.popup` service](documentation/services.md).
 
 The card will be visible only while you're in Edit mode.
 
 Custom popup cards are either local to the current Dashboard view (default) or can be used across all views of the Dashboard. Use the `Popup card is available for use in all views` GUI switch or `popup_card_all_views` optional parameter in Yaml. Using global view custom popup cards allows you to use a sub view to store your custom popup cards for a Dashboard, if that fits your use case.
 
+Using a wide target (label, area, device) and/or popup cards which are global to a view, allows for much customisation of more-info dialog to suit most cases.
```

Comment:
I think it would be good to include a couple of examples to show how this mechanism can be put to use.  Maybe some suggestions on when it might make sense to use an area as a target, for example.  I have some more thoughts on targeting that I'll put in the body of the review.

Reply:
I will update documentation.